source/getting-started/managing-dependencies.md

```diff
@@ -0,0 +1,141 @@
+Ember CLI uses the [Bower](http://bower.io) package manager, making it easy
+to keep your front-end dependencies up to date. NPM is used for development
+dependencies (like addons).
+
+## Bower Configuration
+
+The Bower configuration file, `bower.json`, is located at the root of your Ember
+CLI project, and lists the dependencies for your project. Executing `bower install`
+will install all of the dependencies listed in `bower.json` in one step.
+
+Ember CLI is configured to have git ignore your `bower_components` directory by
+default. Using the Bower configuration file allows collaborators to fork your
+repo and get their dependencies installed locally by executing
+`bower install` themselves.
+
+Ember CLI watches `bower.json` for changes. Thus it reloads your app if you
+install new dependencies via `bower install <dependencies> --save`.
+
+Further documentation about Bower is available at their
+[official documentation page](http://bower.io/).
+
+## Addons
+
+Ember Addons can provide re-usable components to your application or extend
+Ember CLI with new functionality (for example SCSS compilation).
+You can find listings of addons on [Ember Observer](http://emberobserver.com).
+Ember Addons are installed using NPM (eg. `npm install --save-dev ember-cli-sass`).
+
+Addons can automatically add dependencies to your Project's `bower.json` file.
+
+## Compiling Assets
+
+When you're using Bower dependencies that are not included in an Addon, you will
+have to instruct Ember CLI to include your assets in the build. This is done using
+the Asset manifest file `ember-cli-build.js`.
+
+#### Javascript assets exporting a globals
+
+Provide the asset path as the first and only argument.
+
+```ember-cli-build.js
+app.import('bower_components/moment/moment.js');
+```
+
+The globals provided by the asset ( `moment` in the example below can now
+be used in your application without the need to `import` them. You will need to
+tell JSHint that by adding `/* global MY_GLOBAL */` to the files where you're using globals, or
```

Comment:
"to the files where you're using globals" doesn't seem clear enough for me, can you propose an alternative phrasing?